core/src/collections/DynamicObject.cpp

```diff
@@ -184,5 +184,48 @@ namespace ledger {
                 }
             }
         }
+
+        std::shared_ptr<api::DynamicObject> DynamicObject::updateWithConfiguration(const std::shared_ptr<DynamicObject> &configuration) {
```

Comment:
[FAR] We should redo all such method with visitors.

Reply:
Agreed ! 👍